src/documentation/index.html

```diff
@@ -31,7 +31,7 @@
 
     <!--#include virtual="/assets/includes/head-assets.html"-->
 
-    <link href="/assets/v3/css/qg-documentation.css" rel="stylesheet" type="text/css" media="all">
+    <link href="/assets/_project/css/qg-documentation.css" rel="stylesheet" type="text/css" media="all">
```

Comment:
Part of the build process dynamically grabs "assets/_project" and changes them to whatever the version number for the application is set to in the gulp file. Otherwise when you change this number you have to hunt through the templates and change it.